<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>First Qt Project</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2d2d2d;</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QFormLayout" name="formLayout_5">
    <item row="0" column="0" colspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>470</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2d2d2d;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="DeviceInfoPage">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="deviceinfoLabel">
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>deviceinfo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>PN:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Rev:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>OS:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>SN:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Sipariş No:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Ürün Seri No:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #A9A9A9;
border-radius: 4px;
padding: 4px;
background-color: white;
color: black;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit2">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #A9A9A9;
border-radius: 4px;
padding: 4px;
background-color: white;
color: black;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit3">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #A9A9A9;
border-radius: 4px;
padding: 4px;
background-color: white;
color: black;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit4">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #A9A9A9;
border-radius: 4px;
padding: 4px;
background-color: white;
color: black;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit5">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #A9A9A9;
border-radius: 4px;
padding: 4px;
background-color: white;
color: black;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit6">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 2px solid #A9A9A9;
border-radius: 4px;
padding: 4px;
background-color: white;
color: black;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ConfigurationPage">
       <widget class="QLabel" name="configurationLabel">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Configuration Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="UtilitiesPage">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="utilitiesLabel">
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Utilities Page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ConnectionPage">
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
              </property>
              <property name="topMargin">
               <number>7</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="activeportsLabel">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;</string>
                </property>
                <property name="text">
                 <string>Active Ports:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="serialportscomboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>350</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;
</string>
                </property>
                <property name="maxVisibleItems">
                 <number>10</number>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertPolicy::InsertAtBottom</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
                </property>
                <property name="minimumContentsLength">
                 <number>0</number>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="modelColumn">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>40</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">border-color: #2d2d2d;</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <widget class="QPushButton" name="connectportsButton">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>61</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: green;
border-radius: 10px;
color: beige;
font-family: 'Segoe UI', sans-serif;
font-size: 9pt;
font-weight: bold;

</string>
               </property>
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
              <widget class="QPushButton" name="refreshButton">
               <property name="geometry">
                <rect>
                 <x>80</x>
                 <y>10</y>
                 <width>61</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: orange;
border-radius: 10px;
color: beige;
border-radius: 10px;
font-family: 'Segoe UI', sans-serif;
font-size: 9pt;
font-weight: bold;

</string>
               </property>
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>427</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>1080</width>
              <height>720</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2d2d2d;
border: 1px solid #444;
border-radius: 8px;
color: #ffffff;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="configurationButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* Temel Stil */
    background-color: #f57c00;
    color: #212121;
	border: 1px solid rgba(80, 80, 80, 0.6);  /* #505050'nin opak versiyonu */
    
    /* Kenar Yumuşatma */
    border-radius: 8px;  /* Köşeleri belirgin şekilde yuvarlak */
    
    /* Metin &amp; Padding */
    font-family: 'Segoe UI', sans-serif;
    font-size: 12pt;
    font-weight: bold;
    padding: 10px 20px;
    
}

QPushButton:hover {
    background-color: #ff9800;  /* Daha açık turuncu */
}

QPushButton:pressed {
    background-color: #e65100;  /* Daha koyu turuncu */
}</string>
         </property>
         <property name="text">
          <string>Utilities</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="connectionButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* Temel Stil */
    background-color: #f57c00;
    color: #212121;
	border: 1px solid rgba(80, 80, 80, 0.6);  /* #505050'nin opak versiyonu */
    
    /* Kenar Yumuşatma */
    border-radius: 8px;  /* Köşeleri belirgin şekilde yuvarlak */
    
    /* Metin &amp; Padding */
    font-family: 'Segoe UI', sans-serif;
    font-size: 12pt;
    font-weight: bold;
    padding: 10px 20px;
    
}

QPushButton:hover {
    background-color: #ff9800;  /* Daha açık turuncu */
}

QPushButton:pressed {
    background-color: #e65100;  /* Daha koyu turuncu */
}</string>
         </property>
         <property name="text">
          <string>Connection</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="utilitiesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* Temel Stil */
    background-color: #f57c00;
    color: #212121;
	border: 1px solid rgba(80, 80, 80, 0.6);  /* #505050'nin opak versiyonu */
    
    /* Kenar Yumuşatma */
    border-radius: 8px;  /* Köşeleri belirgin şekilde yuvarlak */
    
    /* Metin &amp; Padding */
    font-family: 'Segoe UI', sans-serif;
    font-size: 12pt;
    font-weight: bold;
    padding: 10px 20px;
    
}

QPushButton:hover {
    background-color: #ff9800;  /* Daha açık turuncu */
}

QPushButton:pressed {
    background-color: #e65100;  /* Daha koyu turuncu */
}</string>
         </property>
         <property name="text">
          <string>Configuraiton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="deviceinfoButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* Temel Stil */
    background-color: #f57c00;
    color: #212121;
	border: 1px solid rgba(80, 80, 80, 0.6);  /* #505050'nin opak versiyonu */
    
    /* Kenar Yumuşatma */
    border-radius: 8px;  /* Köşeleri belirgin şekilde yuvarlak */
    
    /* Metin &amp; Padding */
    font-family: 'Segoe UI', sans-serif;
    font-size: 12pt;
    font-weight: bold;
    padding: 10px 20px;
    
}

QPushButton:hover {
    background-color: #ff9800;  /* Daha açık turuncu */
}

QPushButton:pressed {
    background-color: #e65100;  /* Daha koyu turuncu */
}</string>
         </property>
         <property name="text">
          <string>Device Info</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
